src/main/java/games/strategy/engine/data/TripleaHandler.java

```diff
@@ -0,0 +1,393 @@
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_ATTACHMENT_LIST;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_BOOLEAN;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_CONNECTION;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_COST;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_DELEGATE;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_FRONTIER_RULES;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_GAME;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_GAME_PLAY;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_INFO;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_INITIALIZE;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_MAP;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_NUMBER;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_OPTION;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_OWNER_INITIALIZE;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_PLAYER;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_PLAYER_LIST;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_PRODUCTION;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_PRODUCTION_FRONTIER;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_PRODUCTION_RULE;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_PROPERTY;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_PROPERTY_LIST;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_RESOURCE;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_RESOURCE_LIST;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_RESULT;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_SEQUENCE;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_STEP;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_TERRITORY;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_TERRITORY_OWNER;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_UNIT;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_UNIT_INITIALIZE;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_UNIT_LIST;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_UNIT_PLACEMENT;
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_VALUE;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Deque;
+import java.util.LinkedList;
+
+import org.xml.sax.Attributes;
+import org.xml.sax.SAXException;
+import org.xml.sax.SAXParseException;
+import org.xml.sax.helpers.AttributesImpl;
+import org.xml.sax.helpers.DefaultHandler;
+
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.Multimap;
+
+public abstract class TripleaHandler extends DefaultHandler {
```

Comment:
This is the intention.
Because they are not exactly the same.
MapXmlHelper wants to parse the xml into a GUI which edits those values. Tge game parser wants to translate it into a gamedata object which can be used to run the actual game.
Everything which is the same is supposed to be handled inside the abstract Tripleahandler class